spec/07-variables.md

```diff
@@ -254,7 +254,7 @@ echo $colors[100];      // element with offset 100 is still undefined and NULL
                         // is used as substitution value instead. Another
                         // notice is emitted.
 
-$b = &colors[100];      // a VSlot for $b is created which points to the array
+$b = &$colors[100];      // a VSlot for $b is created which points to the array
```

Comment:
Should drop a space from the indentation so comment stays aligned.

Reply:
I don't understand what you mean. Could you be more verbose?

Reply:
You should drop one space after the semicolon so that a comment starts at the same column as line above.